nautobot/ipam/api/views.py

```diff
@@ -140,7 +140,9 @@ def available_prefixes(self, request, pk=None):
         prefix = get_object_or_404(self.queryset, pk=pk)
         if request.method == "POST":
 
-            with cache.lock("available-prefixes", blocking_timeout=5):
+            with cache.lock(
+                "available-prefixes", blocking_timeout=5, timeout=get_settings_or_config("AVAILABLE_IP_PREFIX_TIMEOUT")
```

Comment:
Only need to use `get_settings_or_config` for properties that are alternatively configurable via the Constance (admin-config) UI. As presently implemented, just `settings.AVAILABLE_IP_PREFIX_TIMEOUT` would be more correct.

Reply:
Done